models/review-model.js

```diff
@@ -15,4 +15,45 @@ function fetchReview(id) {
   });
 }
 
-module.exports = { fetchReview };
+function fetchReviewComments(id) {
+  const queryString = `
+  SELECT * FROM comments 
+  WHERE review_id = $1 
+  ORDER BY created_at DESC;
+  `;
+  const value = [id];
```

Comment:
value isnt the best naming really, could maybe just put this array straight in there instead of assign to a new variable?